test/integration/short/pool-simulator-tests.js

```diff
@@ -232,6 +232,7 @@ describe('pool', function () {
 
       const client = new Client({
         contactPoints: cluster.getContactPoints(),
+        localDataCenter: 'dc1',
```

Comment:
its not clear to me why simulacron provisions dc1 as the first dc in one case, and dc0 in another..will see if I can wrap my head around that tomorrow.

Reply:
In some tests we were explicitly provisioning dc0, have since fixed this.